message/message_test.go

```diff
@@ -7,10 +7,11 @@ package message_test
 import "net"
 
 var (
-	mac1, _ = net.ParseMAC("12:34:56:78:90:01")
-	mac2, _ = net.ParseMAC("12:34:56:78:90:02")
-	mac3, _ = net.ParseMAC("12:34:56:78:90:03")
-	mac4, _ = net.ParseMAC("12:34:56:78:90:04")
+	mac1, _         = net.ParseMAC("12:34:56:78:90:01")
+	mac2, _         = net.ParseMAC("12:34:56:78:90:02")
+	mac3, _         = net.ParseMAC("12:34:56:78:90:03")
+	mac4, _         = net.ParseMAC("12:34:56:78:90:04")
+	TsnBridgeMac, _ = net.ParseMAC("12:34:56:78:90:05:67:89")
```

Comment:
This should not be exported. https://go.dev/doc/effective_go#names

```suggestion
	tsnBridgeMac, _ = net.ParseMAC("12:34:56:78:90:05:67:89")
```